### PR TITLE
Add AGPL License

### PR DESCRIPTION
**Pull request by chorman0773**
This proposes to apply the APGL v3+ License to the project.

### CODE REVIEW

**Review by Suya1671 — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
 # Passman
 
-Passman is a Open Source End to End Encrypted Password Manager written in Rust.
\ No newline at end of file
+Passman is a Open Source End to End Encrypted Password Manager written in Rust.
+
+## License Notice
+
```
Comment (kilbouri) [inline README.md:6]: Do we need two lines here?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
```
Comment (kilbouri) [inline README.md:21]: Two lines?

**Review by kilbouri — CHANGES_REQUESTED** [review]

**Review by rdrpenguin04 — APPROVED** [review]
I'll be honest, I wasn't onboard with AGPL at first, but I'm ok with it after hearing the rationale. The main reason is that AGPL will enforce either consistency or transparency between implementations.

File: README.md (hunk 1 above)
Comment (chorman0773) [reply README.md:6]: I like to have a blank line before and after a heading in markdown.

Comment (kilbouri) [reply README.md:6]: Yes, _a_ blank line. There are two. I never remember to select the lines I care about when adding comments, that's on me.

**Review by kilbouri — APPROVED** [review]

**Pull request merged**